docs/api/examples/manage-env.mdx

```diff
@@ -81,12 +81,20 @@ Copy Environment currently copies the following objects:
 
 There is currently no support for ReBAC Role Derivations.
 :::
+:::note GitOps Custom Policies
+When using GitOps and copying an environment to a new target environment, the target environment will also include the custom policies
+under `custom` directory.<br />
+When using GitOps and copying an environment to an existing target environment, the custom policies under `custom`
+directory will be copied only if the conflict strategy is `overwrite`.<br />
+You can modify the custom policies to copy with `custom_policies` key in the request "scope" object.
```

Comment:
this is unclear, maybe change it from note to section and then show an example

Reply:
Which part do you want to extract ? 
What do you mean by section ?

Reply:
Extract all the GitOps explanation to a subsection
### Copy environment that uses GitOps 

When using GitOps and copying an environment to a new target environment, the target environment will also include the custom policies
under `custom` directory.<br />
When using GitOps and copying an environment to an existing target environment, the custom policies under `custom`
directory will be copied only if the conflict strategy is `overwrite`. if you have GitOps in your env and you didn't set the strategy as overwrite it will  ___________<br />

You can modify the custom policies to copy with `custom_policies` key in the request "scope" object.
for example: _____________

Reply:
fixed